resources.html

```diff
@@ -24,6 +24,8 @@
 
 <ul>
 
+<li>All User Guides for <a href="http://downloads.openmicroscopy.org/help/archives/OMERO_User_Guides_5_3_0.zip">Version 5.3.0</a> (Zipped PDFs 159 MB)</li>
```

Comment:
I'm looking at http://help.staging.openmicroscopy.org/resources.html and wondering what to do with it - do you reckon just delete the page completely?